CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- `filter` option for `CadenzaClient#show` and `CadenzaClient#showMap`
```

Comment:
This is an invalid change log: The "### Added" is missing.

Reply:
Fixed.

Reply:
But you didn't _change_ that option, you _added_ it. (See the link at the top of the CHANGELOG for the possible values.)

Reply:
Fixed again.